lib/datastore/transaction.js

```diff
@@ -143,10 +141,10 @@ Transaction.prototype.rollback = function(callback) {
  * Commit the remote transaction and finalize the current transaction instance.
  *
  * @param {function} callback - The callback function.
+ * @private
```

Comment:
Yeah. Maybe our docs with `runInTransaction` should show `done` named as `commit`. Or just explain calling done will perform the commit.

```js
dataset.runInTransaction(function(transaction, commit) {
  // ...
```